src/views/ingresses/actions/hooks/useIngressActions.ts

```diff
@@ -0,0 +1,58 @@
+import { useHistory } from 'react-router';
+
+import { IngressModel, modelToRef } from '@kubevirt-ui/kubevirt-api/console';
+import { IoK8sApiNetworkingV1Ingress } from '@kubevirt-ui/kubevirt-api/kubernetes/models';
+import {
+  Action,
+  useAnnotationsModal,
+  useDeleteModal,
+  useLabelsModal,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { useNetworkingTranslation } from '@utils/hooks/useNetworkingTranslation';
+import { asAccessReview, getName, getNamespace } from '@utils/resources/shared';
+
+type UseIngressActions = (ingress: IoK8sApiNetworkingV1Ingress) => [actions: Action[]];
+
+const useIngressActions: UseIngressActions = (ingress) => {
+  const { t } = useNetworkingTranslation();
+
+  const history = useHistory();
```

Comment:
please check what version of reactrouter are installed, we should move all react-rouer usage to react-router-compat-v5 and use the import same as react router v6, instead of history for example we should use navigate from useNavigate

Reply:
Done!